### PR TITLE
ODIS Update: Expect pre-blinded phone number

### LINKED ISSUE #97: ODIS Receive Blinded Phone Number
Update Komenci API to receive blinded phone number

### PR DESCRIPTION
**Pull request by codyborn**
Update ContractKit version to only perform ODIS request (not blinding within Komenci). Expect pre-blinded number to be passed in API. Note that this is a breaking change and we'll need to update KomenciKit in parallel.

Updated README 
Updated git submodule for celo-monorepo

Closes #97

### CODE REVIEW

File: apps/onboarding/src/dto/DistributedBlindedPepperDto.ts [hunk 1]
```diff
@@ -6,9 +6,8 @@ import {
 } from 'class-validator'
 
 export class DistributedBlindedPepperDto {
-  @IsNotEmpty()
-  @IsPhoneNumber('ZZ')
-  e164Number: string
+  @IsString()
```
Comment (bowd) [inline apps/onboarding/src/dto/DistributedBlindedPepperDto.ts:9]: Not sure about the format of this, but there's also an `@IsHexadecimal()` if that helps.

File: apps/relayer/src/app.controller.spec.ts [hunk 2]
```diff
@@ -111,7 +111,7 @@ describe('AppController', () => {
       })
 
       const input: DistributedBlindedPepperDto = {
-        e164Number: PHONE_NUMBER,
+        blindedPhoneNumber: await BLINDED_PHONE_NUMBER,
```
Comment (bowd) [inline apps/relayer/src/app.controller.spec.ts:114]: Not sure why this `await` is here

**Review by bowd — APPROVED** [review]
:+1:

File: apps/onboarding/src/dto/DistributedBlindedPepperDto.ts (hunk 1 above)
Comment (codyborn) [reply apps/onboarding/src/dto/DistributedBlindedPepperDto.ts:9]: Good call out.. it should be base64 encoded and we have check for that `@IsBase64()`

File: apps/relayer/src/app.controller.spec.ts (hunk 2 above)
Comment (codyborn) [reply apps/relayer/src/app.controller.spec.ts:114]: `getBlindedPhoneNumber` is async now as it's using the implementation in `@celo/contractkit/lib/identity/odis/phone-number-identifier`. This bubbles up through to this function.

Comment (bowd) [reply apps/relayer/src/app.controller.spec.ts:114]: Ah lol that's a Promise. Ok it's fine but there's something slightly off putting about have a Promise stored in a constant. I'd rather have this as a helper function or smth but yeah we can leave it as is, ma bad!

Comment (codyborn) [reply apps/relayer/src/app.controller.spec.ts]: Ah yeah, that is strange looking. Just fixed it.

**Pull request merged**